frontend/public/components/modals/delete-namespace-modal.jsx

```diff
@@ -28,7 +28,7 @@ class DeleteNamespaceModal extends PromiseComponent {
   }
 
   render() {
-    return <form onSubmit={this._submit} name="form">
+    return <form onSubmit={this._submit} name="form" className="modal-content">
       <ModalTitle>Delete {this.props.kind.label}</ModalTitle>
       <ModalBody className="modal-body co-delete-modal">
```

Comment:
Note the passing of `className` here overrides the default classNames in modal.tsx.  As a result, this modal doesn't get the height adjustments.  Is that intentional?